crates/spicepod/Cargo.toml

```diff
@@ -24,7 +24,9 @@ snafu.workspace = true
 tokio.workspace = true
 tracing.workspace = true
 url.workspace = true
+twox-hash = { workspace = true, optional = true }
```

Comment:
I don't think we should feature flag the spicepod crate - it should ideally represent all possible inputs and then we handle those inputs appropriately in the runtime crates.